src/types/inc/viewport.hpp

```diff
@@ -13,15 +13,20 @@ Author(s):
 
 #pragma once
 #include "../../inc/operators.hpp"
+#include "../../inc/til.h"

```

Comment:
if i recall, this one is included by default now!

Reply:
It should be. This shouldn't be needed.